frontend/constants/risks.ts

```diff
@@ -56,6 +56,16 @@ export const RISK_CONSEQUENCES = [...LEVELS_LIST];
 
 export const LEVELS_OF_RISK = [...LEVELS_LIST];
 
+export const MAIN_RISK_TYPE = 'risk';
+export const TREATMENT_RISK_TYPE = 'treatment';
+export const ATTACHMENTS_RISK_TYPE = 'attachments';
+
```

Comment:
@Charence RISK_MITIGATION_STATUSES below (I can't add a comment on that line since it's unchanged...

do we need to rename unmitigated to untreated?

Reply:
You are right, for some reason during saving at some moment I getting undefined value for this variable. I will try to debug this and figure out what causes this